taichi/transforms/statement_replace.cpp

```diff
@@ -5,16 +5,81 @@
 
 TLANG_NAMESPACE_BEGIN
 
+// TODO: rewrite and simplify these classes
+// Replace all usages and remove the statements themselves
+class StatementReplaceAndRemove : public IRVisitor {
```

Comment:
Can we unify this class and `StatementReplace`?

Reply:
Maybe I'll rewrite it in the next PR.